### ISSUE TITLE
JAWS doesn't have ability to read links inside field's description

### ISSUE THREAD

**Issue by Semchik** [FreedomScientific/VFO-standards-support#572]
### Summary

Steps to reproduce:
1. Create HTML page with 'input' tag.
2. Create 'div' element with 'id' attribute.
3. Add some text inside 'div' element.
4. Create child anchor tag inside 'div' after text.
5. Add 'aria-describedby' tag to input, with a value of 'div' element id.
6. Focus 'a' element with Tab.

### Expected result

I expect content of 'div' element to be pronounced by JAWS, then pronounce anchor tag.  

### Actual result

Only anchor tag is pronounced.
 
### Example

https://jsfiddle.net/a1fgkopr/

### Additional Information

I understand, that probably current behavior is correct. But there must be a way to implement expected behavior above, because JAWS user should have ability to focus on link in description and be informed that link is a part of the description.
For now, I only found a workaround wrapping input in 'fieldset'. But we can't use 'fieldset' for each single input.

#### JAWS version and build number

2021.2105.53 ILM
 
#### Operating System and version

Windows 10

#### Browser and version:
Chrome 92.0.4515.159

**Comment by JAWS-test** [comment]
This is not a JAWS error. Every screen reader behaves like this: If a link is focused, it is output. The context of the link can sometimes be determined using special keyboard shortcuts. [WCAG requirement 2.4.4](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html) is related to this.

**Comment by frex65** [comment]
This is indeed expected behaviour. When the user tabs to an anchor link, JAWS announces the link text but not any surrounding text.
